Add 2024 Jenkins elections candidate statements (#7558)
jenkins-infra/jenkins.io: content/blog/2024/10/03/2024-10-03-jenkins-election-candidates.adoc

link:https://www.linkedin.com/in/stefan-spieker-446168161/[LinkedIn],
link:https://contributors.jenkins.io/pages/contributors/stefan-spieker/[Contributor spotlight],
link:https://community.jenkins.io/u/stefanspieker/summary[Jenkins community],
link:https://stackoverflow.com/users/1287864/s-spieker[Stackoverflow]

[[jonesbusy]]
=== link:/blog/authors/jonesbusy/[Valentin Delaye]

I've been a Jenkins user since it's inception in 2011 and contributor since 2018.
As of today I maintain more than 35 plugins.

Starting of this year I got even more involved in the Jenkins community, by participating as a mentor for the Google Summer of Code to work on the Jenkins Plugin Modernizer Tool which is a CLI to update plugins at scale using OpenRewrite.

I'm particularly interested in the "cloud ready" aspect of Jenkins including deployments, packaging (like Jenkinsfile Runner) or distributed storage.
A recent contribution on this topic is the link:https://plugins.jenkins.io/artifactory-artifact-manager/[Artifactory Artifact Manager plugin] that I authored and maintain.

If I'm elected on the Jenkins governance board those will be my subjects of choice.

> Affiliations: Undeclared

Reviewer: ```suggestion
Affiliations: ELCA Cloud Services
```

Profile Links:
link:https://github.com/jonesbusy[GitHub],
link:https://www.linkedin.com/in/valentindelaye/[LinkedIn],
link:https://contributors.jenkins.io/pages/contributors/valentin-delaye/[Contributor spotlight],
link:https://community.jenkins.io/u/jonesbusy/summary[Jenkins community]

[[krisstern]]
=== link:/blog/authors/krisstern/[Kris Stern]

> Kris has been helping out with Jenkins' GSoC participation organization since 2022 and has volunteered to be a GSoC project mentor.

Reviewer 2: ```suggestion
I have been a Jenkins contributor since 2021.
```

> She has participated in GSoC twice as a contributor/student previously in 2019 and 2020, and has been trained academically as an astrophysicist with a PhD in the discipline of observational astronomy obtained from the University of Hong Kong in 2021.

Reviewer 2: ```suggestion
Currently I am a maintainer of four plugins, have been a GSoC org admin and mentor for Jenkins for three years, as well as am a Jenkins Release Team member. 
```

> Professionally, Kris works in the IT sector as a software engineer.

Reviewer 2: ```suggestion
Academically I have been trained as an observational astrophysicist and obtained my PhD degree from the University of Hong Kong (HKU) in 2021.
```

> She has work experiences in Python, C++, Java, JavaScript, TypeScript, HTML, CSS/Sass, JQuery, SQL, and has completed projects in software development in general and specifically in artificial intelligence/deep learning/computer vision, Qt programming, and web development.

Reviewer 2: ```suggestion
I have worked professionally as a software engineer since late 2019, currently working in a AiFi startup.
```

> Kris is passionate about open-source and would like to share this passion with fellow learners.

Reviewer 2: ```suggestion
Currently I am also a part-time MCIT Online student at UPenn. 
I am passionate about open-source and would like to share my love of open-source software with the others. 
```

> Currently, Kris is a part-time MCIT Online student at UPenn.

Reviewer 2: ```suggestion
I feel grateful to be nominated. If elected, I would love to bring with me a startup mindset to help keep Jenkins perpetually youthful and relevant as well as a welcoming community to newcomers. ```

> Affiliations: Undeclared

Reviewer 2: ```suggestion
Affiliations: GAIB.ai
```

[[krisstern]]
=== link:/blog/authors/krisstern/[Kris Stern]

I have been a Jenkins contributor since 2021.
Currently I am a maintainer of four plugins, have been a GSoC org admin and mentor for Jenkins for three years, as well as am a Jenkins Release Team member.
Academically I have been trained as an observational astrophysicist and obtained my PhD degree from the University of Hong Kong (HKU) in 2021.
I have worked professionally as a software engineer since late 2019, currently working in a AiFi startup.
Currently I am also a part-time MCIT Online student at UPenn.
I am passionate about open-source and would like to share my love of open-source software with the others.
I feel grateful to be nominated. If elected, I would love to bring with me a startup mindset to help keep Jenkins perpetually youthful and relevant as well as a welcoming community to newcomers.

Affiliations: GAIB.ai

Profile Links:
link:https://github.com/krisstern[GitHub],
link:https://www.linkedin.com/in/kris-stern/[LinkedIn],
link:https://contributors.jenkins.io/pages/contributors/kris-stern/[Contributor spotlight],
link:https://community.jenkins.io/u/krisstern/summary[Jenkins community]

[[oleg_nenashev]]

> === link:/blog/authors/oleg_nenashev/[Oleg Nenashev]

Reviewer 3: I will need to update the author profile a bit, too

Author: I believe the author profile update was done in:

* #7566

Reviewer 3: Yep

> Jenkins core maintainer, open source software and open hardware advocate.
> Oleg started using Hudson for Hardware/Embedded projects in 2008 and became an active Jenkins contributor in 2012.
> He maintains Jenkinsfile Runner and organizes Jenkins meetups in Switzerland and online.
> 
> Affiliations: Gradle, Inc.

Reviewer 3: ```suggestion
I’ve participated in Jenkins since 2012, including being a core maintainer since 2014,
I was a governance board member from 2019 for two terms (Dec 2019-2023) and also represented Jenkins on the Continuous Delivery Foundation TOC and the board.
During my work on Jenkins, I participated in many projects including JCasC, pluggable storage, and Jenkinsfile Runner. I also created or maintained around 30 plugins for Jenkins.
Many community programs, including GSoC and Hacktoberfest participation, were initially started by me.
In 2022. I took a sabbatical as a core and plugin maintainer due to the war in Ukraine, but I plan to return to the maintainer roles, too.

For me, Jenkins has one of the strongest open source communities, and I am proud to be a part of it.
Should I be elected to the board, my focus will be community growth via onboarding new contributors, improving contributor/developer experience for Jenkins, and facilitating technical partnerships.
The ecosystem evolves, and Jenkins should be changing and growing along with that.
For me, the priorities would be adopting open standards,
integrations with the key projects and services in the cloud-native, CI/CD, and Java space, and continuing Jenkins' evolution as a multi-purpose automation framework.

Affiliations: Gradle Inc., CNCF/CDF, API Neuchatel, Independent
```

Profile Links:
link:https://github.com/oleg-nenashev[GitHub],

> link:https://www.linkedin.com/in/onenashev/[LinkedIn],

Reviewer 3: ```suggestion
link:https://www.linkedin.com/in/onenashev/[LinkedIn],
link:https://sessionize.com/onenashev[Sessionize]
```

I have been a Jenkins contributor since 2021.
Currently I am a maintainer of four plugins, have been a GSoC org admin and mentor for Jenkins for three years, and continue to be a Jenkins Release Team member.
Academically I have been trained as an observational astrophysicist and obtained my PhD degree from the University of Hong Kong (HKU) in 2021.
I have worked professionally as a software engineer since late 2019, currently working in a AiFi startup.
Currently I am also a part-time MCIT Online student at UPenn.
I am passionate about open source and would like to share my love of open-source software with the others.
I feel grateful to be nominated. If elected, I would love to bring with me a startup mindset to help keep Jenkins perpetually youthful and relevant as well as a welcoming community to newcomers.

Affiliations: GAIB.ai

Profile Links:
link:https://github.com/krisstern[GitHub],
link:https://www.linkedin.com/in/kris-stern/[LinkedIn],
link:https://contributors.jenkins.io/pages/contributors/kris-stern/[Contributor spotlight],
link:https://community.jenkins.io/u/krisstern/summary[Jenkins community]

[[oleg_nenashev]]
=== link:/blog/authors/oleg_nenashev/[Oleg Nenashev]

> I’ve participated in Jenkins since 2012, including being a core maintainer since 2014,

Reviewer 4: ```suggestion
I’ve participated in Jenkins since 2012, including being a core maintainer since 2014.
```
Adjusting punctuation

I was a governance board member from 2019 for two terms (Dec 2019-2023) and also represented Jenkins on the Continuous Delivery Foundation TOC and the board.
During my work on Jenkins, I participated in many projects including JCasC, pluggable storage, and Jenkinsfile Runner. I also created or maintained around 30 plugins for Jenkins.
Many community programs, including GSoC and Hacktoberfest participation, were initially started by me.

> In 2022. I took a sabbatical as a core and plugin maintainer due to the war in Ukraine, but I plan to return to the maintainer roles, too.

Reviewer 4: ```suggestion
In 2022, I took a sabbatical as a core and plugin maintainer due to the war in Ukraine, but I plan to return to the maintainer roles.
```
Adjusted punctuation and grammar

For me, Jenkins has one of the strongest open source communities, and I am proud to be a part of it.
Should I be elected to the board, my focus will be community growth via onboarding new contributors, improving contributor/developer experience for Jenkins, and facilitating technical partnerships.

> The ecosystem evolves, and Jenkins should be changing and growing along with that.

Reviewer 4: ```suggestion
The ecosystem evolves and Jenkins should be changing and growing along with that.
```
Adjusting punctuation
